lib/node_modules/@stdlib/stats/base/range/docs/repl.txt

```diff
@@ -1,92 +1,124 @@
+{{alias}}( N, x, strideX )
+
 
-{{alias}}( N, x, stride )
     Computes the range of a strided array.
 
-    The `N` and `stride` parameters determine which elements in `x` are accessed
-    at runtime.
 
-    Indexing is relative to the first index. To introduce an offset, use a typed
-    array view.
+    The `N` and stride parameters determine which elements in the strided 
+    array are accessed at runtime.
+
+
+    Indexing is relative to the first index. To introduce an offset, use a 
+    typed array view.
+
 
     If `N <= 0`, the function returns `NaN`.
 
+
     Parameters
     ----------
+
```

Comment:
Why are you adding all these blank lines? Please remove and follow our REPL text style guide.